app/views/events/participants.html.erb

```diff
@@ -23,7 +24,10 @@
         <% end %>
       </tr>
 
-      <% @participants.each do |participant| %>
+      <%
+      @participants.sort_by! {|l| @event.participant_group_for(l).send(params[:sort]) } if params[:sort]
+      @participants.reverse! if params[:order] == 'descending'
```

Comment:
Oh, hold on, also: the `send(params[:sort])` is insecure since it allows the user to execute any function of the group.